ui/package.json

```diff
@@ -97,7 +96,7 @@
         "redux-devtools-extension": "2.13.9",
         "redux-mock-store": "1.5.4",
         "redux-thunk": "2.4.1",
-        "request": "2.88.2",
+        "axios": "1.4.0",
```

Comment:
Considered using native nodejs functions but ran into connectivity errors that were not easily resolvable.
I believe this current solution using `axios` is appropriate for the time being as it involves very little code change and is a popular [npm package](https://www.npmjs.com/package/axios) that has a very active community supporting it.